client/state/data-layer/wpcom/read/sites/test/index.js

```diff
@@ -0,0 +1,28 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { requestReadSite } from '../';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { requestSite } from 'state/reader/sites/actions';
+
+describe( 'read-sites-site', () => {
+	describe( 'requestReadSite', () => {
+		test( 'should dispatch an http request', () => {
+			const blogId = 123;
+			const action = requestSite( blogId );
+
+			expect( requestReadSite( action ) ).toMatchObject(
+				http(
+					{
+						apiVersion: '1.1',
+						method: 'GET',
+						path: `/read/sites/${ blogId }`,
```

Comment:
why aren't we checking more fields like the query

Reply:
I thought it might result in a brittle test, because if we add a field later, the test will fail. We could import the `fields` file into the test suite too?